test/Altinn.App.Api.Tests/Controllers/ProcessControllerTests.cs

```diff
@@ -0,0 +1,88 @@
+using System.Net;
+using System.Net.Http.Headers;
+using Altinn.App.Api.Tests.Data;
+using Altinn.App.Api.Tests.Utils;
+using FluentAssertions;
+using Microsoft.AspNetCore.Components.Sections;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace Altinn.App.Api.Tests.Controllers
+{
+    public class ProcessControllerTests : ApiTestBase, IClassFixture<WebApplicationFactory<Program>>
+    {
+        public ProcessControllerTests(WebApplicationFactory<Program> factory) : base(factory)
+        {
+        }
+
+        [Fact]
+        public async Task Get_ShouldReturnProcessTasks()
+        {
+            string org = "tdd";
+            string app = "contributer-restriction";
+            int partyId = 500000;
+            Guid instanceId = new Guid("5d9e906b-83ed-44df-85a7-2f104c640bff");
+            HttpClient client = GetRootedClient(org, app);
+
+            TestData.DeleteInstance(org, app, partyId, instanceId);
+            TestData.PrepareInstance(org, app, partyId, instanceId);
+
+            string token = PrincipalUtil.GetToken(1337, 500000, 3);
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+
+            string url = $"/{org}/{app}/instances/{partyId}/{instanceId}/process";
+            HttpResponseMessage response = await client.GetAsync(url);
+            TestData.DeleteInstance(org, app, partyId, instanceId);
+            var content = await response.Content.ReadAsStringAsync();
+
+            response.StatusCode.Should().Be(HttpStatusCode.OK);
+            content.Should().BeEquivalentTo(""" 
```

Comment:
`"string".Should().BeEquivalentTo(...)` gjør nok ikke helt det du vil

```
Asserts that a string is exactly the same as another string, including 
any leading or trailing whitespace, with the exception of the casing.
```

Hvis casing endres her, så vil vi antakelig vite om at vi endrer apet og fikse serialiseringen til å bli lik som før.

Har skrevet slike tester selv før, og veldig irriterende vanskelig å få dem til å kjøre riktig cross platform (`\n` vs `\r\n`, tabs/spaces, indentering ...). Teit av FluentAssertions å ikke tilby `Should().JsonEquivalent()` eller noe sånt, slik at all json formatering og escaping i strings ble ignorert.

Reply:
Yes. Hele testeningen er et work in progress. Vemund skulle hjelpe meg med å se på hvorfor vi får 403 i testen, derav litt eksperimentell kode :sweat_smile: Har konvertert denne tilbake til en draft PR.